.ci/docker/common/install_executorch.sh

```diff
@@ -50,7 +50,7 @@ setup_executorch() {
   pushd executorch
 
   export PYTHON_EXECUTABLE=python
-  export CMAKE_ARGS="-DEXECUTORCH_BUILD_PYBIND=ON -DEXECUTORCH_BUILD_XNNPACK=ON -DEXECUTORCH_BUILD_KERNELS_QUANTIZED=ON"
+  export CMAKE_ARGS="-DEXECUTORCH_BUILD_PYBIND=ON -DEXECUTORCH_BUILD_XNNPACK=ON -DEXECUTORCH_BUILD_KERNELS_QUANTIZED=ON -DEXECUTORCH_BUILD_TESTS=ON"
```

Comment:
A follow-up AI here I think is to create a short script on ExecuTorch CI to do this step https://github.com/pytorch/executorch/blob/main/.ci/scripts/unittest-linux.sh#L20-L26 so that we could just call it here instead of copying the logic over, which could get out of sync when ET is updated